org.civicrm.reqangular/src/common/modules/uib-tabset.js

```diff
@@ -0,0 +1,18 @@
+define([
+  'common/angular',
+  'common/angularBootstrap',
+  'common/modules/templates'
+], function (angular) {
+  'use strict';
+
+  return angular.module("common.uibTabset", [])
```

Comment:
you are adding an entire new angular module specifically to simply alter the html of a single directive, it is not necessary (let's not overcomplicate things, especially if we don't even have yet a consistent approach with decorators, see `common.angularDate`) nor scalable (because you called the module with a very specific name)

Please put this in `common.directives`, if will need improvements then we'll take it from there